content/en/docs/concepts/cluster-administration/priority-and-fairness.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Priority and Fairness in Handling Requests
+content_template: templates/concept
+weight: 20
+---
+
+
+{{% capture overview %}}
+
+{{< feature-state state="alpha" >}}
+
+The control plane of Kubernetes is itself a resource (broadly
+construed) to which access is managed.  In particular, each apiserver
+applies some prioritization and fairness in its handling of requests.
+This includes not only kube-apiservers but aggregated apiservers too.
+Prioritization and fairness applies only to requests that are not
+_long running_.  Long running requests are WATCHes and EXECUTEs.
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+## Status
+
+This feature is named "RequestManagement" and is alpha in
+release 1.16.  As an alpha feature, it is disabled by default.  When
+enabled it replaces a request filter that enforces simple aggregate
+concurrency limits with this feature, which can be seen as a more
+general version of that earlier functionality.
+
+## Outline
```

Comment:
This text might belong in the overview section.

Reply:
👍

Reply:
The outline has to be a single paragraph, right?

Reply:
Not sure what you mean. Is this page supposed to be a blog or content page?

Reply:
This is a page for the CONCEPTS section of the documentation, and I thought I read somewhere that the overview of a CONCEPTS page has to be a single paragraph.

Reply:
It ought to be short, but if you need to use 2 paragraphs then I think that can be justified. A 2 paragraph overview seems better to me than 2 separate overview-like sections.